tests/Unit/Bundle/Maker/Factory/NoPersistanceObjectsAutoCompleterTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Zenstruck\Foundry\Tests\Unit\Bundle\Maker\Factory;
+
+use PHPUnit\Framework\TestCase;
+use Zenstruck\Foundry\Bundle\Maker\Factory\NoPersistanceObjectsAutoCompleter;
+
+final class NoPersistanceObjectsAutoCompleterTest extends TestCase
```

Comment:
I don't think (but I may be wrong) that it is possible to test this autocompletion behavior, even with `CommandCompletionTester`, this is why I used a unit test